distribution/docs/src/main/jdocs/content/_quickstart/quickstart-ingesting-contents.adoc

```diff
@@ -0,0 +1,33 @@
+:title: Ingesting (Quick Start)
+:type: quickStart
+:status: published
+:summary: Ingest sample data.
+:order: 02
+
+=== Ingesting (Quick Start)
+
+Now that ${branding} has been configured, ingest some sample data to demonstrate search capabilities.
+
+This is one way to ingest into the catalog, for a complete list of the different methods, see <<_ingesting_data,Ingesting Data>>.
+
+==== Ingesting Sample Data
+
+. Download a sample valid https://codice.atlassian.net/wiki/download/attachments/1179756/geojson_valid.json?version=1&modificationDate=1368249436010&api=v2[GeoJson file here].
+. Navigate in the browser to the Search UI at ${secure_url}/search.
+. Click the *upload* icon in the upper right corner.
+. Select the file to upload.
+
+[NOTE]
+====
+XML metadata for text searching is not automatically generated from GeoJson fields.
+====
+
+Querying from the Search UI (${secure_url}/search) will return the record for the file ingested:
```

Comment:
This is only true is the search UI user has a clearance equal to or higher than the system user. For example, if the admin choose the "SIPR" or "JWICS" profile, during installation, the search UI would not return any results. The search UI user would have to logout as "admin" and log back in as "localhost". 

This problem is easy to spot. If the search UI says something like "returned 0 of 1 hits", then you know you have a permissions issue.